beps/0003-metadata-on-relations/README.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Support Metadata on Relations
+status: provisional
+authors:
+  - '@sennyeya'
+project-areas:
+  - catalog
```

Comment:
Could you add an entry to CODEOWNERS as well? 🙏

Reply:
Added, went ahead and put both myself and the catalog maintainers, happy to update if needed.